src/brutus-api/brutus_api/tests/nlc_test.py

```diff
@@ -0,0 +1,100 @@
+import unittest
+
```

Comment:
Personally, I like to separate my imports into chunks from top to bottom: standard libraries, third-party libraries, local project, local module. Also PEP8 requires two blank lines above any top-level class or function definition.